website/docs/r/monitor_inhibition_rule.md

```diff
@@ -0,0 +1,93 @@
+---
+subcategory: "Sysdig Monitor"
+layout: "sysdig"
+page_title: "Sysdig: sysdig_monitor_inhibition_rule"
+description: |-
+  Creates a Sysdig Monitor Inhibition Rule.
+---
+
+# Resource: sysdig_monitor_inhibition_rule
+
+Creates a Sysdig Monitor Inhibition Rule.
+
+-> **Note:** Sysdig Terraform Provider is under rapid development at this point. If you experience any issue or discrepancy while using it, please make sure you have the latest version. If the issue persists, or you have a Feature Request to support an additional set of resources, please open a [new issue](https://github.com/sysdiglabs/terraform-provider-sysdig/issues/new) in the GitHub repository.
+
+## Example Usage
+
+```terraform
+resource "sysdig_monitor_inhibition_rule" "sample" {
+  name = "Example Inhibition Rule"
+  description = "Example description"
+  enabled = true
+
+  source_matchers {
+    label_name = "alertname"
+    operator = "EQUALS"
+    value = "networkAlert"
+  }
+
+  source_matchers {
+    label_name = "device_type"
+    operator = "EQUALS"
+    value = "firewall"
+  }
+
+  target_matchers {
+    label_name = "device_type"
+    operator = "REGEXP_MATCHES"
+    value = ".*server.*"
+  }
+
+  equal = ["kube_cluster_name"]
+}
+```
+
+## Argument Reference
+
+* `name` - (Optional) The name of the Inhibition Rule. If provided, it must be unique.
+
+* `description` - (Optional) The description of the Inhibition Rule.
+
+* `enabled` - (Optional) Whether to enable the Inhibition Rule. Default: `true`.
+
+* `equal` - (Optional) List of labels that must have an equal value in the source and target alert for the inhibition to take effect.
+
+### `source_matchers`
+
+List of source matchers for which one or more alerts have to exist for the inhibition to take effect.
```

Comment:
This closely resembles the OSS docs description but I think we can aim for even more clarity. 

> Source Matchers define the label combinations that an alert occurrence must match for the inhibition rule to apply. When an alert occurrence satisfies this criteria, it activates the inhibition rule, which can then suppress notifications for target alerts.

Reply:
ok i added the suggestion.